improver/lightning.py

```diff
@@ -204,3 +204,220 @@ def latitude_to_threshold(
         rescale(latitude, (50.0, 10), (midlatitude, tropics), clip=True),
         rescale(latitude, (-50.0, -10), (midlatitude, tropics), clip=True),
     )
+
+
+class LightningMultivariateProbability(PostProcessingPlugin):
```

Comment:
This isn't a definitive recommendation but related to [this comment](https://github.com/metoppv/improver/pull/1983#issuecomment-1992736096), I think that it would be reasonable to name this `LightningMultivariateProbability_USAF2024` where USAF2024 is effectively the reference in lieu of a citable reference. Therefore, I would think that an algorithm taken from e.g. Bright et al., 2005 might be named `LightningMultivariateProbability_Bright2005`. I think that this naming convention is OK for now, although we're planning to have some more internal discussions about this.

Reply:
I'm ok with this renaming and have changed the name throughout the codebase.